easybuild/easyconfigs/o/OpenFold/OpenFold-1.0.0-foss-2021a-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,91 @@
+easyblock = 'PythonBundle'
+
+name = 'OpenFold'
+version = '1.0.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://github.com/aqlaboratory/openfold'
+description = "A faithful PyTorch reproduction of DeepMind's AlphaFold 2"
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+builddependencies = [
+    ('CMake', '3.20.1'),  # required to build ninja
```

Comment:
This comment is a bit confusing, maybe change it to `# required to build ninja Python package (included as extension)`

Reply:
Ok